stock_quant_manual_assign/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+Manual assignment of quants
+===========================
+
+With this module, user can change manually the automatic selection of quants
+
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```

Comment:
But, this is from the template in https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst
So maybe someone should add what you are telling there :wink: